tabular/tests/conftest.py

```diff
@@ -19,3 +26,118 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
+
+
+class DatasetLoaderHelper:
+    @staticmethod
+    def load_dataset(name: str, directory_prefix: str = './datasets/'):
+        dataset_info = DatasetLoaderHelper.dataset_info_dict()[name]
+        train_file = dataset_info.pop('train_file', 'train_data.csv')
+        test_file = dataset_info.pop('test_file', 'test_data.csv')
+        name_inner = dataset_info.pop('name')
+        url = dataset_info.pop('url', None)
+        train_data, test_data = DatasetLoaderHelper.load_data(
+            directory_prefix=directory_prefix,
+            train_file=train_file,
+            test_file=test_file,
+            name=name_inner,
+            url=url,
+        )
+
+        return train_data, test_data, dataset_info
+
+    @staticmethod
+    def dataset_info_dict():
+        dataset_dict = dict(
```

Comment:
nit: can be just a constant instead of method

Reply:
Updated